javascript/integration-tests/LockDealOrder.spec.ts

```diff
@@ -0,0 +1,127 @@
+// Copyright 2022 Gluwa, Inc. & contributors
+// SPDX-License-Identifier: The Unlicense
+
+import { Guid } from 'js-guid';
+
+import { ApiPromise, Keyring, WsProvider } from '@polkadot/api';
+import { KeyringPair } from '@polkadot/keyring/types';
+import type { Balance } from '@polkadot/types/interfaces';
+
+import { Blockchain, DealOrderId, LoanTerms } from 'credal-js/lib/model';
+import { signLoanParams } from 'credal-js/lib/extrinsics/register-deal-order';
+
+import { POINT_01_CTC } from '../src/constants';
+import { randomEthAddress } from '../src/utils';
+import * as testUtils from './test-utils';
+
+describe('LockDealOrder', (): void => {
+    let api: ApiPromise;
+    let borrower: KeyringPair;
+    let lender: KeyringPair;
+    let dealOrderId: DealOrderId;
+
+    const blockchain: Blockchain = 'Ethereum';
+    const expirationBlock = 10_000;
+    const loanTerms: LoanTerms = {
+        amount: BigInt(1_000),
+        interestRate: {
+            ratePerPeriod: 100,
+            decimals: 4,
+            period: {
+                secs: 60 * 60 * 24,
+                nanos: 0,
+            },
+        },
+        termLength: {
+            secs: 60 * 60 * 24 * 30,
+            nanos: 0,
+        },
+    };
+
+    beforeEach(async () => {
+        process.env.NODE_ENV = 'test';
+
+        const provider = new WsProvider('ws://127.0.0.1:9944');
+        api = await ApiPromise.create({ provider });
+        const keyring = new Keyring({ type: 'sr25519' });
+
+        lender = keyring.addFromUri('//Alice', { name: 'Alice' });
+        const lenderWallet = randomEthAddress();
+        const lenderRegAddr = await testUtils.registerAddress(api, lenderWallet.address, blockchain, lender);
+
+        borrower = keyring.addFromUri('//Bob', { name: 'Bob' });
+        const borrowerWallet = randomEthAddress();
+        const borrowerRegAddr = await testUtils.registerAddress(api, borrowerWallet.address, blockchain, borrower);
+
+        const askGuid = Guid.newGuid();
+        const bidGuid = Guid.newGuid();
+        const signedParams = signLoanParams(api, borrower, expirationBlock, askGuid, bidGuid, loanTerms);
+
+        const result = await testUtils.registerDealOrder(
+            api,
+            lenderRegAddr.addressId,
+            borrowerRegAddr.addressId,
+            loanTerms,
+            expirationBlock,
+            askGuid,
+            bidGuid,
+            borrower.publicKey,
+            signedParams,
+            lender,
+        );
+        dealOrderId = result.dealOrder.dealOrderId;
+
+        // lender is 'Alice', who should also be sudoSigner for dev network
+        await testUtils.setupAuthority(api, lender);
+        const [transferKind, txHash] = await testUtils.prepareEthTransfer(
+            lenderWallet,
+            borrowerWallet,
+            dealOrderId,
+            loanTerms,
+        );
```

Comment:
I get a consistent `Transfer verification timed out` for this one, while FundDealOrder.spec.ts reports PASS. Not sure how to debug!